steward/src/commands/cosmos_to_eth.rs

```diff
@@ -10,12 +10,33 @@ use std::{process::exit, time::Duration};
 
 const TIMEOUT: Duration = Duration::from_secs(60);
 
-/// Send Cosmos to the Eth chain
+/// Send Cosmos to the Eth chain.
 #[derive(Command, Debug, Default, Parser)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Send Cosmos token to Eth chain.\n This command sends Cosmos token to the Eth chain via the Gravity bridge. \n This command takes the Gravity denom, tx amount, Cosmos keyname, Eth destination, number of times \n transaction should be made and if the transaction should be made immediately or wait for the next \n batch."
+)]
 pub struct CosmosToEthCmd {
+    /// Gravity denom, takes a String.
     #[clap(short, long)]
-    pub args: Vec<String>,
+    gravity_denom: String,
 
+    /// Tx amount.
+    #[clap(short, long)]
+    amount: String,
+
+    /// Cosmos keyname, takes a String.
+    #[clap(short, long)]
+    cosmos_key: String,
+
+    /// H160 Ethereum address
+    #[clap(short, long)]
+    eth_dest: String,
+
+    /// The number of times transactions should repeat itself, default is 1.
+    #[clap(short, long, default_value = "1")]
+    times: String,
+
+    /// Boolean, True if you want to wait until someone requests a batch for this token type and False if you want to request a batch to push transaction along immediately.
```

Comment:
We might want to consider changing how this flag works, since a double negative is kind of confusing, i.e. `flag_no_batch` being `false` means send a batch. Perhaps changing this to `request_batch` and having `true` send the request would be easier to understand.

Reply:
please can you checkout if what I did solves the problem